nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/source/SecretsDirConfigSource.java

```diff
@@ -43,12 +43,10 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
+import java.nio.file.*;
```

Comment:
Personally, I would avoid using wildcard imports - however, I'll leave it to a consensus

Reply:
Oh that was IntelliJ automagic. Happy to switch back again.

Reply:
Done with 74288cb